components/Panel/VotePanel/VotePanel.tsx

```diff
@@ -24,29 +25,49 @@ export function VotePanel({ content }: Props) {
     options,
   } = content;
 
-  const hasResults = Boolean(results?.length);
-
-  const tabs = [
-    {
-      title: "Result",
-      content: (
-        <Result
-          decodedIdentifier={decodedIdentifier}
-          participation={participation}
-          results={results}
-          options={options}
-        />
-      ),
-    },
+  const { data: discussion, isFetching: discussionLoading } = useVoteDiscussion(
```

Comment:
fetch this data here so that we can load it so long as soon as the user opens the panel